testing/btest/Baseline/bifs.records_fields/out

```diff
@@ -48,3 +48,9 @@ F
 [a] = [type_name=count, log=F, value=<uninitialized>, default_val=<uninitialized>],
 [d] = [type_name=string, log=T, value=<uninitialized>, default_val=<uninitialized>]
 }
+{
+[a] = [type_name=set[double], log=F, value=<uninitialized>, default_val=<uninitialized>],
+[b] = [type_name=set[double,string], log=F, value=<uninitialized>, default_val=<uninitialized>],
+[c] = [type_name=table[double,string] of string, log=F, value=<uninitialized>, default_val=<uninitialized>],
+[d] = [type_name=vector[string], log=F, value=<uninitialized>, default_val=<uninitialized>]
+}
```

Comment:
This baseline is failing for CI (and also for me when running `btest -d` manually) because the elements are out of order: https://travis-ci.com/zeek/zeek/builds/136118908

Did you manually create the baseline or use `btest -U` to do it?  The later should take care of putting things in the canonical order -- the order of container elements depends on a random seed and that's fixed when running via `btest`, but differs from run to run of just normal command-line `zeek`.  Let me know if you general `btest` workflow questions.

Reply:
Yes, I created the baseline manually (and somehow missed that it was failing). 

Didn't know about btest -U, just used it and updated the baseline.